Resources/Prototypes/Entities/Mobs/Species/reptilian.yml

```
@@ -9,7 +9,7 @@
     species: Reptilian
     hideLayersOnEquip:
     - Snout
-    - HeadTop
+#    - HeadTop # WWDP - Because visible horns its cool
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)